frontend/src/components/input/PasswordInput.tsx

```
@@ -0,0 +1,70 @@
+  requiredAlert = false,
+  requiredField = false,
+}) => {
+  const [type, setType] = useState<'password' | 'text'>('password');
+  return (
+    <>
+      {label ? (
+        <label
+          className="text-sm font-medium leading-6 text-gray-300"
+          htmlFor={id}
+        >
+          {label + ' '}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)